sdk/ml/azure-ai-ml/azure/ai/ml/_local_endpoints/azureml_image_context.py

```diff
@@ -79,12 +80,13 @@ def __init__(
                     }
                 }
             )
+            # Set the directory containing scoring script as AML_APP_ROOT/working directory
             self._environment[
                 LocalEndpointConstants.ENVVAR_KEY_AML_APP_ROOT
-            ] = docker_code_mount_path  # ie. /var/azureml-app/onlinescoring
+            ] = os.path.join(docker_code_mount_path, os.path.dirname(yaml_code_scoring_script_file_name))  # ie. /var/azureml-app/onlinescoring
```

Comment:
The example path should be updated?

Reply:
It is assumed to be correct as we can see the entry script example is just score.py.